internal/xds/translator/local_ratelimit.go

```diff
@@ -201,56 +199,66 @@ func buildRouteLocalRateLimits(local *ir.LocalRateLimit) (
 
 		// HeaderMatches
 		for mIdx, match := range rule.HeaderMatches {
+			var action *routev3.RateLimit_Action
```

Comment:
curious, if any of this code can be deduped with a function
https://github.com/envoyproxy/gateway/blob/5b6a35f3e6b6b31ad7479a9ceb5159b77cd68f7a/internal/xds/translator/ratelimit.go#L166

Reply:
Ha, may look into later, the rate limit code is pretty complex.